aesm-client/src/imp/windows.rs

```diff
@@ -168,4 +168,20 @@ impl AesmClient {
         }
         return Ok(licence_token);
     }
+
+    pub fn get_supported_att_key_ids(&self) -> Result<AesmKeyIds> {
```

Comment:
I prefer doing `#[cfg(not(windows))]` on the functions you add in src/lib.rs

Reply:
thanks